partner_tier_validation/data/tier_definition.xml

```diff
@@ -9,4 +9,15 @@
             name="definition_domain"
         >["&amp;",["parent_id","=",False],"|",["active","=",True],["active","=",False]]</field>
     </record>
```

Comment:
@patrickrwilson This rule is known to cause issues, whenever Odoo automatically creates a new contact out of an email address. I still think is is not wise to include it.

Reply:
I will remove this rule then.

Reply:
I'm fine with making this the disabled rule and the new one the default rule with your suggested changes to the description,

Reply:
eh, I just decided to remove it like you originally suggested.